trio/_ssl.py

```diff
@@ -207,15 +214,19 @@ class NeedHandshakeError(Exception):
 
     """
 
+    __slots__ = ()
+
 
 class _Once:
-    def __init__(self, afn, *args):
+    __slots__ = ("_afn", "_args", "started", "_done")
```

Comment:
Maybe you missing my initial comment: I think this and the `__slots__` above should probably be moved to your other PR.

Reply:
Moved to #2754

Reply:
Oops, I think you forgot to remove the changes from here :P

Reply:
Yep, forgot to push the commit